src/languages/es.js

```diff
@@ -365,11 +365,14 @@ export default {
         removeContactMethod: 'Eliminar método de contacto',
         removeAreYouSure: '¿Estás seguro de que quieres eliminar este método de contacto? Esta acción no se puede deshacer.',
         resendMagicCode: 'Reenviar código mágico',
+        failedNewContact: 'Error al añadir este método de contacto',
         genericFailureMessages: {
             requestContactMethodValidateCode: 'No se ha podido enviar un nuevo código mágico. Espera un rato y vuelve a intentarlo.',
             validateSecondaryLogin: 'No se ha podido validar el método de contacto con el código mágico provisto. Solicita un nuevo código y vuelve a intentarlo.',
             deleteContactMethod: 'No se ha podido eliminar el método de contacto. Por favor contacta con Concierge para obtener ayuda.',
+            addContactMethod: 'No se ha podido añadir el método de contacto. Ponte en contacto con Conserjería para obtener ayuda.',
```

Comment:
+1 I am gonna guess "Concierge" is part of our brand identity which sort of transcends language, but maybe @Expensify/marketing has the final word on that...

Reply:
i think we should keep it as Concierge if we're treating it like a proper noun (e.g. a name or a brand name) but I put it in here to discuss in case anyone feels otherwise https://expensify.slack.com/archives/C21FRDWCV/p1680657688103779

Reply:
fixed by reusing another existing key `common.conciergeHelp`

Reply:
Thank you all for your thoughts!